testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/ServerAuthenticationTestCase.java

```diff
@@ -131,6 +116,7 @@ public void testDisableLocalAuthAndStartServers() throws Exception {
         // now restart the slave HC, but don't restart the servers and verify they've reconnected afterwards
         reloadHostController(master.getDomainClient(), "slave", false);
         slave.awaitHostController(System.currentTimeMillis());
+        waitUntilState(master.getDomainClient(), "slave", "main-three", "STARTED");
         verifyServerStarted(master.getDomainClient(), "slave", "main-three");
```

Comment:
This shouldn't be necessary since AIUI waitUntilState will fail if the server doesn't get to the right state.

Can you change waitUntilState so it's not waiting a full 1 second between tries? That seems like a long time. Maybe 100ms? 250ms?